src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -448,10 +452,24 @@ function WorkspaceMembersPage(props) {
                         {data.length > 0 ? (
                             <View style={[styles.w100, styles.mt4, styles.flex1]}>
                                 <View style={[styles.peopleRow, styles.ph5, styles.pb3]}>
-                                    <Checkbox
-                                        isChecked={!_.isEmpty(removableMembers) && _.every(_.keys(removableMembers), (accountID) => _.contains(selectedEmployees, Number(accountID)))}
+                                    <PressableWithFeedback
+                                        disabled={_.isEmpty(removableMembers)}
                                         onPress={() => toggleAllUsers(removableMembers)}
-                                    />
+                                        accessibilityRole="checkbox"
+                                        accessibilityState={{
+                                            checked: !_.isEmpty(removableMembers) && _.every(_.keys(removableMembers), (accountID) => _.contains(selectedEmployees, Number(accountID))),
+                                        }}
+                                        accessibilityLabel={props.translate('workspace.people.selectAll')}
+                                        // disable hover dimming
```

Comment:
- Add a new line before the comment
- Capitalize the comment
- Explain **why** we're doing this

Reply:
I think we should remove those comments since they just say what the code also says, and don't add any additional value